content/docs/cucumber/api.md

```diff
@@ -969,6 +969,8 @@ A tag expression is an *infix boolean expression*. Below are some examples:
 | `@smoke and @fast`   |                   Scenarios tagged with both `@smoke` and `@fast` |
 | `@gui or @database`  |                Scenarios tagged with either `@gui` or `@database` |
 
+Note: Using a `,` to separate tags in expressions such as `@tag1,@tag2` will cause the entire expression to be parsed as a single tag, potentially leading to unintended outcomes. It's required to separate the tags with space or operators like `and` `or` and `not`.
```

Comment:
This is legacy and we shouldn't be documenting this. Please use the documentation above.

Reply:
@luke-hill I think the purpose was to explain people that there was a different syntax for tag expressions in the past and that they should upgrade theirs.

Reply:
In previous areas we've actively removed legacy information, so if we're going down that route (removing legacy info), I'd rather keep consistent than not.

But yes notwithstanding that, if something indicated that this isn't advised then yes I figure that could be ok